src/media_changer.py

```diff
@@ -5929,7 +5938,7 @@ def mtx_dismount(self, volume, drive, media_type="", view_first=1):
     # numbers are not negative one then there is either a bug in this
     # function or multiple tapes that have the same label in the
     # library.
-    def locate_volume(self, vol):
+    def locate_volume(self, vol, ignore_addresses=[]):
         Trace.trace(ACTION_LOG_LEVEL, ' looking for volume %s status_valid %s' % (vol, self.status_valid))
```

Comment:
I think this is python pitfall number one - never use mutable default argument


(https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

replace with : 
```
def locate_volume(self, vol, ignore_addresses=None):
```

Reply:
Ah, thanks - PyCharm has helped me fix enough of these throughout the codebase I should know this by now. Will change now

Reply:
Resolved in [0e7292d](https://github.com/Enstore-org/enstore/pull/118/commits/0e7292dc4adc19dbe5cd37103fb2c74333e162e7)